Makefile

```diff
@@ -0,0 +1,2 @@
+install-deps:
+	pip install packaging
```

Comment:
make is kind of a smell -- also it's really easy to get wrong (this should have for example `.PHONY: install-deps` and it should probably build a virtualenv)

the requirements for this live inside `./docker/...`

the better way to run the tests and use the utilities is through `tox`, which is kinda like a python-specific command running / automation tool that _replaces_ make

Reply:
tox also automates setting up an environment via `tox --devenv venv`